site/content/contribute/more-info/server/style-guide.md

```diff
@@ -159,6 +159,74 @@ func OtherFunction() {
 }
 ```
 
+### Errors
+
+Always add proper context to errors.
+
+#### Include the user input for validation errors.
+
+For example:
+
+`return fmt.Errorf("invalid export type, must be one of: csv, actiance, globalrelay")` does not include what was the input value entered by the user. It needs to be included.
+
+Another example:
```

Comment:
Should we pick just one example?

Reply:
More examples don't hurt IMO. If we choose the diff format, it takes up less space and makes it clearer. Let's see what Carrie thinks.

Reply:
I don't have concerns about having multiple examples. Let's then ensure they are all up to the standards and provide clear guidance. See https://github.com/mattermost/mattermost-developer-documentation/pull/1455#discussion_r2126359252.